### DISCUSSION TITLE
Registration deadline

### DISCUSSION THREAD

**Discussion by zhejiangzhoulei** [ds4dm/ml4co-competition#46] [General]
Hi all,
I'm wondering what the deadline is for submitting the pre-registration form?

Best,
Lei

**Comment by gasse** [comment]
Hi @zhejiangzhoulei ,

We hadn't really fixed a deadline for team registrations as we wanted to allow teams to join during the competition, but we ought to have one. We will update the website with a precise date and make an announcement soon.

Meanwhile, I can tell you that the registration deadline will be a couple of weeks before the end of the competition. New participants can still join in August, September and early October. 

Thanks for asking !

Best,
Maxime

**Reply by zhejiangzhoulei** [reply]
Hi @gasse ,
Thank you for your response :)
